internal/configs/transportserver.go

```diff
@@ -59,14 +59,14 @@ func generateTransportServerConfig(transportServerEx *TransportServerEx, listene
 		proxyTimeout = transportServerEx.TransportServer.Spec.SessionParameters.Timeout
 	}
 
-	statusZone := ""
+	serverSnippets := generateSnippets(snippetsEnabled, transportServerEx.TransportServer.Spec.ServerSnippets, []string{})
```

Comment:
because of the recent change, we no longer need to pass  `snippetsEnabled` to `generateTransportServerConfig`, because after validation `transportServerEx.TransportServer.Spec.ServerSnippets` is (1) either empty when snippets are disabled or (2) it is either empty or set to something when snippets are enabled. This means there is no way snippetsEnabled is false and transportServerEx.TransportServer.Spec.ServerSnippets is set to something.

Perhaps we can have a special function for TransportServer, like
```
serverSnippets := generateSnippetsForTransportServer(transportServerEx.TransportServer.Spec.ServerSnippets, []string{})
```

or simply:
```
	var serverSnippets []string
	if transportServerEx.TransportServer.Spec.ServerSnippets != "" {
		serverSnippets = []string{strings.Split(transportServerEx.TransportServer.Spec.ServerSnippets, "\n")}
	}
```

that will also remove the need for the unit test `TestGenerateTransportServerConfigForTCPSnippetsNotEnabled`

Reply:
I'm leaving the function as is and passing in false, removing the enable flag from the ts config.